core/domain/collection_jobs_one_off.py

```diff
@@ -61,7 +61,7 @@ def map(item):
         try:
             collection.validate(strict=False)
         except Exception as e:
-            logging.error(
+            logging.exception(
```

Comment:
Do we expect developers not to use logging.error in the future? (if yes, are we planning to have a linter check for it?)

Reply:
The decision is up to the `exception` also includes the stack trace as part of the log.